recipes/ndtypes/run_test.bat

```diff
@@ -0,0 +1 @@
+cd "%RECIPE_DIR%" && "%PYTHON%" test_ndtypes.py
```

Comment:
Please add this file to `meta.yaml`'s [`test/files`]( https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-files ). Then `cd`-ing won't be necessary. Can also add `python test_ndtypes.py` in [`test/commands`]( https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands ). Should eliminate the need for this and `run_test.sh`.